models/Profile/Event.js

```diff
@@ -34,6 +34,14 @@ const EventSchema = new Schema({
   price: {
     startingFrom: Number,
   },
+  location:{
+    road:String,
+    city:String,
+    state:String,
+    country:String,
+    lat:Number,
+    lon:Number
+  }
```

Comment:
Project code formatting, I think Prettier would change it to this...

```suggestion
  location: {
    road: String,
    city: String,
    state: String,
    country: String,
    lat: Number,
    lon: Number,
  },
```